AnkiDroid/src/main/java/com/ichi2/libanki/template/Replacement.kt

```diff
@@ -45,7 +42,7 @@ class Replacement(
     @VisibleForTesting
     constructor(key: String, vararg filters: String?) : this(
         key,
-        Arrays.asList<String?>(*filters),
+        filters.map { it!! },
```

Comment:
```suggestion
    constructor(key: String, vararg filters: String) : this(
        key,
        filters.toList(),
```
the callers (only tests) always send non-null args, so the param can be made non-null